conf/distro/include/rust_security_flags.inc

```diff
@@ -1,4 +1,6 @@
 # Build errors with PIE options enabled
 SECURITY_CFLAGS_pn-rust-native = "${SECURITY_NO_PIE_CFLAGS}"
+SECURITY_CFLAGS_pn-rust-cross-${TARGET_ARCH} = "${SECURITY_NO_PIE_CFLAGS}"
 SECURITY_CFLAGS_pn-rust-cross = "${SECURITY_NO_PIE_CFLAGS}"
```

Comment:
That's true; I will remove that.  I'll wait to see if there are any more changes.